noxfile.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
This should be reverted to 2019 since it's not a new file.

Reply:
Changed.